Code/Engine/GameEngine/StateMachine/Implementation/StateMachineBuiltins.cpp

```diff
@@ -0,0 +1,184 @@
+#include <GameEngine/GameEnginePCH.h>
+
+#include <Foundation/IO/TypeVersionContext.h>
+#include <GameEngine/StateMachine/Implementation/StateMachineBuiltins.h>
+
```

Comment:
Currently you only have the send message state. I could imagine a "log" state might be useful at least for debugging. Also in a lot of cases I would guess a state that itself does nothing is already sufficient. Both "do nothing" and "log something" could be handled by the same state, though.

Reply:
Maybe, but not as part of this PR